data/data-pipeline/data_pipeline/etl/score/etl_score.py

```diff
@@ -464,62 +390,62 @@ def transform(self) -> None:
             "Score E",
             "Poverty (Less than 200% of federal poverty line)",
         ]:
-            self.df[f"{score_field}{self.PERCENTILE_FIELD_SUFFIX}"] = self.df[
+            df[f"{score_field}{self.PERCENTILE_FIELD_SUFFIX}"] = df[
                 score_field
             ].rank(pct=True)
 
             for threshold in [0.25, 0.3, 0.35, 0.4]:
                 fraction_converted_to_percent = int(100 * threshold)
-                self.df[
+                df[
                     f"{score_field} (top {fraction_converted_to_percent}th percentile)"
                 ] = (
-                    self.df[f"{score_field}{self.PERCENTILE_FIELD_SUFFIX}"]
+                    df[f"{score_field}{self.PERCENTILE_FIELD_SUFFIX}"]
                     >= 1 - threshold
                 )
+        return df
 
-        # Now for binary (non index) scores.
-
-        # Calculate "Score F", which uses "either/or" thresholds.
+    def _add_score_f(self, df: pd.DataFrame) -> pd.DataFrame:
+        logger.info("Adding Score F")
         ami_and_high_school_field_name = "Low AMI, Low HS graduation"
         meets_socio_field_name = "Meets socioeconomic criteria"
         meets_burden_field_name = "Meets burden criteria"
 
-        self.df[ami_and_high_school_field_name] = (
-            self.df[self.MEDIAN_INCOME_AS_PERCENT_OF_STATE_FIELD_NAME] < 0.80
-        ) & (self.df[self.HIGH_SCHOOL_FIELD_NAME] > 0.2)
+        df[ami_and_high_school_field_name] = (
+            df[self.MEDIAN_INCOME_AS_PERCENT_OF_STATE_FIELD_NAME] < 0.80
+        ) & (df[self.HIGH_SCHOOL_FIELD_NAME] > 0.2)
```

Comment:
maybe make this clearer that this is for individuals with less than high school degre